packages/image/src/providers/vercel.js

```diff
@@ -0,0 +1,18 @@
+import { set_param, relative_url } from './utils';
+
+// https://vercel.com/docs/concepts/image-optimization
+
+/** @type {import("types").GetURL<{ quality?: number }>} */
+export function getURL({ src, width, options }) {
```

Comment:
height is in the interface, but unused in this and the other providers. should it be implemented?